auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -20,15 +22,26 @@ import '../server/request_handler.dart';
 class CheckPullRequest extends RequestHandler {
   CheckPullRequest({
     required Config config,
+    this.pubsub = const PubSub(),
   }) : super(config: config);
 
-  Future<Response> get(Request request) async {
-    //TODO(Kristin): Change the way to get this PR later according to the real situation, https://github.com/flutter/flutter/issues/99720
-    final String rawBody = await request.readAsString();
-    final body = json.decode(rawBody) as Map<String, dynamic>;
-    final PullRequest pullRequest = PullRequest.fromJson(body['pull_request']);
-    final RepositorySlug slug = pullRequest.base!.repo!.slug();
+  final PubSub pubsub;
+
+  Future<Response> get() async {
+    final pub.PullResponse pullResponse = await pubsub.pull(1, 'auto-submit-queue-sub');
```

Comment:
We want to do some large number like 100

Reply:
Sure, made the number larger.